core/base/Dockerfile

```diff
@@ -61,16 +61,32 @@ RUN set -euxo pipefail \
         mkdir -p /root/.cargo/registry/index && \
         git clone --bare https://github.com/rust-lang/crates.io-index.git /root/.cargo/registry/index/github.com-1285ae84e5963aae \
     ; pip install -r requirements-${MAILU_DEPS}.txt \
-    ; apk del -r .build-deps \
     ; rm -rf /root/.cargo /tmp/*.pem \
   ; } \
   ; rm -rf /root/.cache
 
+ARG SNUFFLEUPAGUS_VERSION=0.8.3
```

Comment:
If you need some of the build deps here you should join the two run commands into one.

Reply:
Done in 63a12d98